lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -73,22 +73,34 @@ window.getListOfAudits = function() {
 };
 
 window.saveAudits = function(audits) {
-  const storage = {};
-  storage[STORAGE_KEY] = audits;
+  const listOfAudits = window.getListOfAudits().map(aggregation => aggregation.name);
+  let storage = {};
+  storage[STORAGE_KEY] = {};
+
+  window.getListOfAudits().forEach(audit => {
+    storage[STORAGE_KEY][audit.name] = audits.indexOf(audit.name) > -1;
+  });
 
   chrome.storage.local.set(storage);
 };
 
 window.fetchAudits = function() {
   return new Promise(resolve => {
-    chrome.storage.local.get(STORAGE_KEY, function(result) {
-      let audits = result[STORAGE_KEY];
+    chrome.storage.local.get(STORAGE_KEY, result => {
+      const audits = result[STORAGE_KEY];
 
-      if (!audits) {
-        audits = window.getListOfAudits().map(aggregation => aggregation.name);
-      }
+      // create list of default audits
+      const listOfAudits = window.getListOfAudits().map(aggregation => aggregation.name);
```

Comment:
window.getListOfAudits() gives me the array with all available audits (checkboxes in the extension). I fetch the name for every bit and than make an object that has the audit as key and value as a boolean. For the list of default audits i put it on true and let the user overwrite it with whatever is in storage.

This gives us the possibility to add more audits without changing storage key with a version in the name.

when looking at that piece of code I can reduce a few lines.